uiowa/uiowa
# Commencement program archive + session term updates

Resolves #7577 

# How to test

```
ddev blt ds --site commencemnt.uiowa.edu && ddev drush @commencement.local uli admin/structure/taxonomy/manage/session/add
```
- Add some sessions, including programs
- Confirm that year list gets populated in reverse chronological order
- Confirm that session title is generated from season + year
- Add a page, save, layout, add program archive block
- Confirm view displays all the sessions you added
- Confirm program link displays as "View program" instead of program file name
